### PR TITLE
move all hints how to fix "leapp preupgrade" issues under one point

### PR DESCRIPTION
**Pull request by evgeni**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Comment by evgeni** [comment]
@bangelic @maximiliankolb this is the reordering I mentioned in #1571.
My idea is to tell the user a more structured story how to behave when they run into "problems" during `leapp preupgrade` (I am putting "problems" in quotes, as those are mostly legit configurations that can't be upgraded and need manual intervention).
My thought is to tell:

* run the command, it will most probably report issues, so please fix those.
* here is a list of the most common fixes we know of
* oh, if it tells you to do X (network name change), you need to do this *ADDITIONAL* work as a Foreman/Satellite specialty.

**Comment by evgeni** [comment]
and of course this broke formatting…

File: guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc [hunk 1]
```diff
@@ -146,49 +146,45 @@ endif::[]
...
 +
 If this happens, do the following to clean up packages that cannot automatically upgrade (`rubygem(thor)` and `rubygem(railties)` in the example above):
-
 +
 ----
 # yum remove rubygem-thor rubygem-railties
 ----
++
 endif::[]
+If `leapp preupgrade` inhibits the upgrade with *Unsupported network configuration* because there are multiple legacy named network interfaces, follow the instructions shown by Leapp to rename the interfaces, followed by an installer run to reconfigure {Project} to use the new interface names:
```
Comment (evgeni) [inline guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc:172]: this section is indented too much, and I have no idea how to fix it

**Comment by maximiliankolb** [comment]
> My thought is to tell:
> 
> * run the command, it will most probably report issues, so please fix those.
> * here is a list of the most common fixes we know of
> * oh, if it tells you to do X (network name change), you need to do this _ADDITIONAL_ work as a Foreman/Satellite specialty.

Yes, I like this structure. It also reminds me of your talk at Foreman Community Demo a while ago. 👍

**Comment by evgeni** [comment]
Okay, I got the rendering the way I wanted, please review! :)

**Review by maximiliankolb — APPROVED** [review]

**Comment by evgeni** [comment]
26eb6d196..1458d566d  3.1 -> 3.1
   5dc078725..adb34178e  3.2 -> 3.2
   38576233c..b07126902  3.3 -> 3.3
   7706ad015..6ba872bdf  3.4 -> 3.4

**Pull request merged**